pkg/interceptors/interceptors.go

```diff
@@ -17,27 +17,83 @@ limitations under the License.
 package interceptors
 
 import (
+	"errors"
+	"fmt"
 	"net/http"
+	"time"
 
 	triggersv1 "github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
 )
 
-// Interceptor is the interface that all interceptors implement.
-type Interceptor interface {
-	ExecuteTrigger(req *http.Request) (*http.Response, error)
+// WebhookSecretStore provides cached lookups of k8s secrets, backed by a Reflector.
+type WebhookSecretStore interface {
+	Get(sr triggersv1.SecretRef) ([]byte, error)
+}
+
+type webhookSecretStore struct {
```

Comment:
To address your question, maybe just `secretStore` would work

Reply:
Yes, which of course was what it was when I first wrote this struct def out 😅